assets/scss/settings/_settings.scss

```diff
@@ -0,0 +1 @@
+$govuk-page-width: 1200px !default;
```

Comment:
Is this a change from the current theme default? If so, I'd suggest breaking this out into a separate commit for clarity.